### DISCUSSION TITLE
Lists features are disappointing especially with starred projects only

### DISCUSSION THREAD

**Discussion by katiestapleton** [community/community#100001] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Starring and adding to a list should remain as different functionalities. In general, a list is for categorization.  I see starring as marking important, move to top of list, or similar. Lists should be applicable to any repo, starred or not. For me, its the old repos that need to be categorized, not my primary repos. 

Right not, the list is an invisible tag with almost no benefits. There is no filters related to lists (find/sort/view only), its not directly visible, lists must be starred, and list groups can't be deleted. I realize I could be wrong; if that's the case then the available features need a better UX/UI . It has potential, but needs to be reworked.

**Comment by datpaitoon17** [comment]
Kotkotkot151996@gmail.com​

**Reply by datpaitoon17** [reply]
@datpaitoon17

**Comment by Danyalkasiri** [comment]
****

**Comment by seanbethard** [comment]
me: Alright, let's create some lists!
me: [creates list]
me: The add to list option is under Starred? That's weird.
me: [adds project to list]
GitHub: [adds project to stars]
me: kbye!

**Comment by katiestapleton** [comment, marked as answer]
This is intended as a feedback, I do not need an answer directly. Since I keep getting notifications for off-topic posts, I am closing it up. It's still an issue

**Comment by Roojsz88** [comment]
http://bit.ly/42GVFH4

**Comment by Khnsimeon29** [comment]
https://hCaptcha.com/?r=1c2cc5f736d1

**Comment by robbertakse** [comment]
> https://hCaptcha.com/?r=1c2cc5f736d1

**Comment by robbertakse** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Starring and adding to a list should remain as different functionalities. In general, a list is for categorization.  I see starring as marking important, move to top of list, or similar. Lists should be applicable to any repo, starred or not. For me, its the old repos that need to be categorized, not my primary repos. 
> 
> Right not, the list is an invisible tag with almost no benefits. There is no filters related to lists (find/sort/view only), its not directly visible, lists must be starred, and list groups can't be deleted. I realize I could be wrong; if that's the case then the available features need a better UX/UI . It has potential, but needs to be reworked.
